static_files/el/bor/config.toml

```diff
@@ -2,20 +2,18 @@ identity = "{{.node_name}}"
 chain = "{{.config_folder_path}}/genesis.json"
 datadir = "{{.data_folder_path}}"
 verbosity = {{.log_level_to_int}}
-{{if eq .kind "stateless"}}
+
+{{if .el_bor_sync_with_witness}}
```

Comment:
Apologies, didn't see the condition here. The flag `el_bor_sync_with_witness` shouldn't be used for setting `syncmode = "stateless"`. I think we will still need a new parameter (maybe `syncmode`?) in param.yml. In bor, there are four types of sync mode, `full`, `snap`, `archive`, and `stateless`. Any node should be able to configure its sync mode freely. The default value will be `full` if not specified.

Reply:
Got it! I made a change so that any node can configure its sync mode: #192 which I also merged in this PR - both should be ready for review now